pkg/core/interop/context.go

```diff
@@ -405,7 +405,7 @@ func (ic *Context) IsHardforkEnabled(hf config.Hardfork) bool {
 	if ok {
 		return ic.BlockHeight() >= height
 	}
-	return len(ic.Hardforks) == 0 // Enable each hard-fork by default.
+	return true // Enable each hard-fork by default.
```

Comment:
> Previous logic was exactly right

I understand that, but this behavior differs from the C# node. In C# node if you don't have the exact hardfork specified in the config, then the hardfork is treated as enabled irrespectively to other hardforks. And I've changed our behaviour because if NeoC# node operator is going to use NeoGo, then it may miss this difference which will lead to unexpected node behaviour.

Are you sure we'd like to preserve that difference with C# config style?

Reply:
Let's discuss this in neo-project/neo then. I'd say it's not correct. Currently I can have a config with
```
hardforks:
    aspidochelone: 100500
```
Then I upgrade to a new node and expect things to work the same way, but suddenly I get `basilisk` enabled which breaks my node.

Reply:
Removed.